sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -233,7 +236,7 @@ void WinHttpTransport::CreateConnectionHandle(
   // Specify an HTTP server.
   // This function always operates synchronously.
   handleManager->m_connectionHandle = WinHttpConnect(
-      handleManager->m_sessionHandle,
+      m_sessionHandle,
```

Comment:
Can the session die or something after a while?  expire or anything which would require re-creating the session again for the adapter?  I don't see a way right now to support this.

Reply:
> Can the session die or something after a while? expire or anything which would require re-creating the session again for the adapter?

Good question. The default session timeout is 60 seconds.

> If a TCP connection request takes longer than this time-out value, the request is canceled. The default timeout is 60 seconds. When you are attempting to connect to multiple IP addresses for a single host (a multihomed host), the timeout limit is for each individual connection.

We could disable the timeout by setting it to infinite (0xFFFFFFFF).